besu/src/main/java/org/hyperledger/besu/cli/options/unstable/NetworkingOptions.java

```diff
@@ -30,6 +30,7 @@ public class NetworkingOptions implements CLIOptions<NetworkingConfiguration> {
   private final String CHECK_MAINTAINED_CONNECTIONS_FREQUENCY_FLAG =
       "--Xp2p-check-maintained-connections-frequency";
   private final String DNS_DISCOVERY_SERVER_OVERRIDE_FLAG = "--Xp2p-dns-discovery-server";
+  private final String DISCOVERY_PROTOCOL_V5_ENABLED = "--Xv5-disc-enabled";
```

Comment:
non-blocking feedback - if we're not exactly matching what geth calls this option, I think I would prefer 'discovery' over 'disc'

Reply:
Agree, I just changed the option name